server/datastore/mysql/apple_mdm.go

```diff
@@ -5372,3 +5377,31 @@ func (ds *Datastore) CountABMTokensWithTermsExpired(ctx context.Context) (int, e
 	}
 	return count, nil
 }
+
+func (ds *Datastore) GetABMTokenOrgNamesForTeam(ctx context.Context, teamID *uint) ([]string, error) {
```

Comment:
Nit: I found the implementation surprising as the name was somewhat misleading to me - it's not the ABM org names for a team associated with that ABM token (e.g. `where macos_default_team_id = ? OR ios_default_team_Id = ? ...`), it's the ABM org names for hosts that are part of this team.

I don't think the name necessarily needs to be changed (it still makes sense!), I think just a comment explaining what it does without having to look at the query would be good enough!

Reply:
+1 on this, I got confused during our slack discussion because of the same thing. It implies multiple ABM tokens per team, which in a sense is true but not in the way you think of it!